openbsd-games.db

```diff
@@ -3,7 +3,7 @@ Cover	AaaaaA_for_the_Awesome_Cover.jpg
 Engine
 Setup
 Runtime	HumblePlay
-Store	https://www.humblebundle.com/store/aaaaaaaaaaaaaaaaaaaaaaaaa-for-the-awesome
+Store	https://www.humblebundle.com/store/aaaaaaaaaaaaaaaaaaaaaaaaa-for-the-awesome https://store.steampowered.com/app/15560
```

Comment:
The humblebundle purchase comes with HumblePlay, meaning it can be played in the browser. The Steam store link is intentionally left out because buying the game there will not offer any way to play it on OpenBSD.

Reply:
Ah, I wrongly assumed that games were similarly distributed across stores. I wasted your time on this. Sorry for that.